scenes/quests/lore_quests/quest_001/3_stealth_level/stealth_level.tscn

```diff
@@ -377,6 +386,7 @@ metadata/_edit_lock_ = true
 [node name="ScreenOverlay" type="CanvasLayer" parent="."]
 
 [node name="HUD" parent="." instance=ExtResource("10_idy4y")]
+visible = false
```

Comment:
This change is probably not intended. Makes the HUD invisible during the Stealth game. I think that the HUD should be visible in all 3 minigames. Or am I missing something?
```suggestion
```

Reply:
you're right!, it's fixed now 👌🏾